config/webpack/production.js

```diff
@@ -6,8 +6,9 @@ const CompressionPlugin = require('compression-webpack-plugin');
 const sharedConfig = require('./shared.js');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const OfflinePlugin = require('offline-plugin');
-const { env, publicPath } = require('./configuration.js');
+const { publicPath } = require('./configuration.js');
 const path = require('path');
+const { URL } = require('url');
```

Comment:
WHATWG URL API is supported from nodejs 8.0 (as non-experimental), but our Dockerfile (and package.json) uses nodejs 6.x for now.

We need to use legacy URL API instead or upgrade those requirements.

Reply:
-> #7665